packages/react-components/src/components/styles/VideoTile.styles.ts

```diff
@@ -86,30 +87,27 @@ export const displayNameStyle: IStyle = {
 /**
  * @private
  */
-export const iconContainerStyle: IStyle = {
-  height: '100%',
+export const iconContainerStyle = (theme: Theme): IStyle => ({
+  margin: 'auto',
   alignItems: 'center',
   '& svg': {
-    display: 'block'
+    display: 'block',
+    color: theme.palette.neutralPrimary
   }
-};
+});
 
 /**
  * @private
  */
-export const participantStateStringStyles = (showLabel: boolean): IStyle => {
+export const participantStateStringStyles = (theme: Theme): IStyle => {
```

Comment:
Text seems off centered compared to the participants name
![image](https://user-images.githubusercontent.com/97130533/188002741-695ee414-2b77-4686-8b83-b71083391602.png)

Reply:
Looks like the previous snapshots are also not aligned.
I will approve this PR as this bug was not introduced with this PR.